lib/src/widgets/chat/audio/ds_audio_message_bubble.widget.dart

```diff
@@ -109,8 +119,36 @@ class _DSAudioMessageBubbleState extends State<DSAudioMessageBubble>
       }
     });
 
-    await _controller.player
-        .setAudioSource(AudioSource.uri(Uri.parse(widget.uri)));
+    Platform.isIOS && widget.audioType.contains('ogg')
+        ? await _transcoder()
+        : await _controller.player
+            .setAudioSource(AudioSource.uri(Uri.parse(widget.uri)));
+  }
+
+  Future<void> _transcoder() async {
+    final inputFileName = 'AUDIO-${widget.uniqueId}.ogg';
+
+    final inputFilePath = await DSFileService.download(
+      widget.uri,
+      inputFileName,
+    );
+
+    final temporaryPath = (await getTemporaryDirectory()).path;
+    final outputFile = File("$temporaryPath/AUDIO-${widget.uniqueId}.mp3");
+
+    if (await outputFile.exists()) {
+      await _controller.player.setFilePath(outputFile.path);
+      return;
```

Comment:
Instead of using a return statement, I'd use an else statement to improve readablity of conditional scenarios.

Reply:
Done.